### ISSUE TITLE
In excel, if in browse mode, you can't arrow around a chart.

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#6509]
Make a basic chart.
Enter browse mode.
Press nvda+f7.
Select your chart.
Arrow around.
Expected behavior: You move through the chart.
Result: You move back to where you were and explore the actual worksheet.

**Comment by jcsteh** [comment]
Confirmed.

P2 because this is a reasonable use case and could leave users unable to access charts. The work-around of switching to focus mode first is not intuitive.

**Comment by Adriani90** [comment]
@derekriemer, @jcsteh I cannot reproduce this issue with NVDA 2019.1 RC1 and Office 2010 and 2016 / 365. Are you still able to reproduce it?

**Comment by jcsteh** [comment]
I no longer have regular access to Microsoft Excel, as I don't use it and thus don't have a license, so I can't test this.

**Comment by bdorer** [comment]
I'm not able to reproduce this with Excel 2013 nvda latest Alpha-Snapshot..

Am 19.03.2019 um 03:25 schrieb James Teh:

**Comment by Adriani90** [comment]
Closing as works for me.

**Comment by Adriani90** [comment]
Thanks Jamie and Bernd for replying.
